### ISSUE TITLE
Finetune gate network after pruning

### ISSUE THREAD

**Issue by delock** [CerebrasResearch/reap#3]
Does finetune gate network after pruning helps increase accuracy, instead of only normalize gated value?  I'm wondering whether this could further improve accuracy, and gate network is usually quite small, thus finetune them would have low cost.

**Comment by mklasby** [comment]
Hi @delock,
For this work we only considered the one-shot pruning / merging case. We are hoping to explore expert compression with fine-tuning approaches in future work. Based on our analysis, your idea to fine-tune just the router is well motivated for expert merging in particular to help restore router/expert coordination. 
Cheers,
-Mike
